tslint.json

```diff
@@ -73,7 +73,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
Why remove this?

Reply:
that's all done as part of the Angular 10 upgrade:
https://github.com/angular/angular-cli/issues/17117

Reply:
Ok, cool.